.gitignore

```diff
@@ -19,7 +19,8 @@
 web/themes/custom/mytheme/dist/javascript.js
 /web/sites/default/settings.local.php
 /web/sites/default/services.local.yml
-web/themes/custom/mytheme/dist/
+/web/themes/custom/mytheme/dist/
+/web/modules/custom/drupal_bootcamp_weather
```

Comment:
I did not get it.

Reply:
This is my first module, I deleted it in php storm and had a problem. Then I fixed problem and decide to  create a new module, these add to ignore...